physical/aerospike/aerospike.go

```diff
@@ -0,0 +1,251 @@
+package aerospike
+
+import (
+	"context"
+	"crypto/sha256"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	aero "github.com/aerospike/aerospike-client-go"
+	log "github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/vault/sdk/helper/strutil"
+	"github.com/hashicorp/vault/sdk/physical"
+)
+
+const (
+	keyBin   = "keyBin"
+	valueBin = "valueBin"
+
+	defaultNamespace = "test"
+
+	defaultHost = "127.0.0.1"
```

Comment:
Nit: Since this is assigned as the default hostname, can we rename this to `defaultHostname`?